packages/components/tooltip/src/tooltip.js

```diff
@@ -226,8 +226,31 @@ Tooltip.propTypes = {
   /**
    * Horizontal size limit of the tooltip.
    */
-  horizontalConstraint: PropTypes.oneOf(['xs', 's', 'm', 'l', 'xl', 'scale'])
-    .isRequired,
+  horizontalConstraint: PropTypes.oneOf([
+    'xs',
+    's',
+    'm',
+    'l',
+    'xl',
+    1,
+    2,
+    3,
+    4,
+    5,
+    6,
+    7,
+    8,
+    9,
+    10,
+    11,
+    12,
+    13,
+    14,
+    15,
+    16,
+    'scale',
+    'auto',
+  ]).isRequired,
```

Comment:
Is this required?

Reply:
I guess it doesn't need to be, since the style defaults to the `scale` constraint anyway.